pkgs/development/python-modules/pandas/default.nix

```diff
@@ -123,19 +129,17 @@ in buildPythonPackage rec {
     echo "#!${runtimeShell}" > pbpaste
     chmod a+x pbcopy pbpaste
     export PATH=$(pwd):$PATH
-  '' + ''
-    LC_ALL="en_US.UTF-8" py.test $out/${python.sitePackages}/pandas --skip-slow --skip-network -k "$disabledTests"
-    runHook postCheck
   '';
 
-  meta = {
+  meta = with stdenv.lib; {
     # https://github.com/pandas-dev/pandas/issues/14866
     # pandas devs are no longer testing i686 so safer to assume it's broken
     broken = stdenv.isi686;
     homepage = "https://pandas.pydata.org/";
+    changelog = "https://pandas.pydata.org/docs/whatsnew/index.html";
```

Comment:
Wasn't even aware we track these. Just read up in the manual that it "may use expansion to refer to the correct changelog". So we could use:" https://pandas.pydata.org/docs/whatsnew/v${version}.html" here.

Reply:
In general, my preference order of operations when adding changelogs to existing packages has been:
1. Generic user-facing changelog (less likely to break), e.g. on readthedocs/whichever site
2. Version-specific link, user-facing
3. GitHub release, populated with version number
4. Some other changelog if I can find it.
Hence I chose 1 here.